scripts/travis-ci/create-release-tarball.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -e
+
+# This script will create a release package suitable for use on production
+# instances. This package excludes all tests and development tools.
+
+# Define paths.
+PROJECT_ROOT=$(pwd)
+RELEASE_PATH=${PROJECT_ROOT}/oe_theme
+
+# Get the release number.
+VERSION=$(git describe --tags)
+if [ $? -ne 0 ]; then
+  echo "No release tag associated with the current HEAD."
+  exit 1
+fi
+
+cd ${PROJECT_ROOT}
+
+# Create release folder.
+mkdir ${RELEASE_PATH}
+
+# Undo vandalism to composer.json.
+git checkout composer.json
+rm composer.lock
+
+# Install production dependencies.
+# Todo: Not needed if we exclude the vendor folder.
+sudo chmod -R a+w build/
```

Comment:
This code block downloads the production dependencies so they can be included in the archive. I put this here for completeness and because it is similar to how Drupal core is packaged.

However, I personally think having the dependencies inside the theme is not needed. Since we are using a composer based workflow in all our projects the dependencies will be automatically downloaded. I don't even know if the old Composer Manager workflow with separate vendor folders per project is still supported in Drupal 8.

Let's discuss this, and we can decide to keep it or remove it.

Reply:
Yeah we should remove the vendor folder, we will have Composer taking care of that for us.